lib/dfe/analytics/event.rb

```diff
@@ -126,15 +126,15 @@ def hash_to_kv_pairs(hash)
       end
 
       def anonymised_user_agent_and_ip(rack_request)
-        DfE::Analytics.pseudonymise(rack_request.user_agent.to_s + rack_request.remote_ip.to_s) if rack_request.remote_ip.present?
+        DfE::Analytics.anonymise(rack_request.user_agent.to_s + rack_request.remote_ip.to_s) if rack_request.remote_ip.present?
```

Comment:
@asatwal I left in this method but reverted the naming to anonymise. This was there prior to your addition of pseudonymise_web_request_user_id, although that's when the naming changed to pseudonymise. I can remove this too and the associated tests but kept it in for time being as it seemed useful if is actually being used

Reply:
@ericaporter I'd suggest we call it something that is aligned with the algorithm. If it's doing a one way hash I'd say pseudonymise; if it's overwriting that part of the string with (say) REDACTED then I'd go anonymise.

This is significant because pseudonymised data stays PII under GDPR; anonymised data doesn't.

Reply:
I would say anonymise is OK as this user agent + ip is not really PII.